<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1817</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 240, 255);</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-color: rgb(229, 240, 255);
color: rgb(39, 154, 255);</string>
   </property>
   <property name="text">
    <string>疫情防控</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>431</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
   </property>
   <property name="title">
    <string>功能区：</string>
   </property>
   <widget class="QPushButton" name="photo_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/picture/btn_photo_normal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
    </property>
    <property name="title">
     <string>模型预测：</string>
    </property>
    <widget class="QTextEdit" name="textage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
     </property>
     <property name="text">
      <string>是否匹配戴口罩：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_mask">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_age">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
     </property>
     <property name="text">
      <string>年龄预测：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_fever">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fever">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
     </property>
     <property name="text">
      <string>发烧预测：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
     </property>
     <property name="text">
      <string>体温输入：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="photo_yc">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(39, 154, 255);
font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>预测</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="camera_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/picture/btn_off.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_camera_open">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
    </property>
    <property name="text">
     <string>摄像头开关：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_camera_photo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
    </property>
    <property name="text">
     <string>拍照开关：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="photo_btn_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/picture/btn_photo_normal.png);
border-image: url(:/newPrefix/picture/btn_photo_normal1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_upload">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
    </property>
    <property name="text">
     <string>上报云端：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zhushi">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>（注意：发烧为布尔值1，未发烧为0）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>640</y>
     <width>731</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
   </property>
   <property name="title">
    <string>各年龄段正常体温范围对照表：</string>
   </property>
   <widget class="QLabel" name="label_zhushi_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(85, 0, 255);</string>
    </property>
    <property name="text">
     <string>0-2岁：36.4-38.0℃           3-10岁：36.1-37.8℃     11-65岁：35.9-37.5℃        &gt;65岁：35.8-37.4℃</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_zhushi_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>该表用于对发烧的预测，以提高发烧判别的准确性</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>731</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
   </property>
   <property name="title">
    <string>视频区：</string>
   </property>
   <widget class="QLabel" name="label_show_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="photo_sx">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>680</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(39, 154, 255);
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>120</y>
     <width>531</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(39, 154, 255);</string>
   </property>
   <property name="title">
    <string>拍照区：</string>
   </property>
   <widget class="QLabel" name="label_show_photo_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../YQ_demo.qrc"/>
 </resources>
 <connections/>
</ui>
